00102-lib64.patch

```diff
@@ -65,30 +65,30 @@ index b9bbfe5..2a5f29c 100644
  
  _INSTALL_SCHEMES = {
      'posix_prefix': {
--        'stdlib': '{installed_base}/lib/python{py_version_short}',
--        'platstdlib': '{platbase}/lib/python{py_version_short}',
-+        'stdlib': '{installed_base}/lib64/python{py_version_short}',
-+        'platstdlib': '{platbase}/lib64/python{py_version_short}',
-         'purelib': '{base}/lib/python{py_version_short}/site-packages',
--        'platlib': '{platbase}/lib/python{py_version_short}/site-packages',
-+        'platlib': '{platbase}/lib64/python{py_version_short}/site-packages',
+-        'stdlib': '{installed_base}/lib/platform-python{py_version_short}',
+-        'platstdlib': '{platbase}/lib/platform-python{py_version_short}',
++        'stdlib': '{installed_base}/lib64/platform-python{py_version_short}',
++        'platstdlib': '{platbase}/lib64/platform-python{py_version_short}',
+         'purelib': '{base}/lib/platform-python{py_version_short}/site-packages',
+-        'platlib': '{platbase}/lib/platform-python{py_version_short}/site-packages',
++        'platlib': '{platbase}/lib64/platform-python{py_version_short}/site-packages',
          'include':
-             '{installed_base}/include/python{py_version_short}{abiflags}',
+             '{installed_base}/include/platform-python{py_version_short}{abiflags}',
          'platinclude':
 @@ -61,10 +61,10 @@ _INSTALL_SCHEMES = {
          'data': '{userbase}',
          },
      'posix_user': {
--        'stdlib': '{userbase}/lib/python{py_version_short}',
--        'platstdlib': '{userbase}/lib/python{py_version_short}',
-+        'stdlib': '{userbase}/lib64/python{py_version_short}',
-+        'platstdlib': '{userbase}/lib64/python{py_version_short}',
-         'purelib': '{userbase}/lib/python{py_version_short}/site-packages',
--        'platlib': '{userbase}/lib/python{py_version_short}/site-packages',
-+        'platlib': '{userbase}/lib64/python{py_version_short}/site-packages',
-         'include': '{userbase}/include/python{py_version_short}',
-         'scripts': '{userbase}/bin',
-         'data': '{userbase}',
+-        'stdlib': '{installed_base}/lib/platform-python{py_version_short}',
+-        'platstdlib': '{platbase}/lib/platform-python{py_version_short}',
++        'stdlib': '{installed_base}/lib64/platform-python{py_version_short}',
++        'platstdlib': '{platbase}/lib64/platform-python{py_version_short}',
+         'purelib': '{base}/lib/platform-python{py_version_short}/site-packages',
+-        'platlib': '{platbase}/lib/platform-python{py_version_short}/site-packages',
++        'platlib': '{platbase}/lib64/platform-python{py_version_short}/site-packages',
+         'include': '{installed_base}/include/platform-python{py_version_short}{abiflags}',
+         'scripts': '{base}/bin',
+         'data': '{base}',
```

Comment:
`posix_user` is config for installing Python modules into a user's home (`~/.local`). That's not supported for platform-python, so `posix_user` was changed to match `posix_prefix`.